lib/microservices/registrations.rb

```diff
@@ -48,6 +52,21 @@ def self.registration_connection
       )
     end
 
+    # rubocop:disable Metrics/ParameterLists
+    def self.add_registration(competition_id, user_id, event_ids, comment, competing_status, current_user)
+      response = self.registration_connection.post(self.add_registration_path(competition_id)) do |req|
+        req.body = { competition_id: competition_id,
+                     user_id: user_id,
+                     event_ids: event_ids,
+                     comment: comment,
+                     competing_status: competing_status,
+                     current_user: current_user }.to_json
```

Comment:
Is the `to_json` actually necessary here? I feel like we already configured the JSON modules to auto-encode both request and response

Reply:
I just tested and it looks like it doesn't